src/pcp.rs

```diff
@@ -476,12 +483,20 @@ impl<F: FieldElement> GadgetCallOnly<F> for QueryShimGadget<F> {
 /// The output of `query`, the verifier message generated for a proof.
 #[derive(Debug)]
 pub struct Verifier<F: FieldElement> {
-    /// Output of the validity circuit.
-    v: F,
-    /// The proof polynomial evluated at `r`.
-    p_at_r: F,
-    /// The intermediate proof polynomials evaluated at `r`.
-    f_at_r: Vec<F>,
+    data: Vec<F>,
```

Comment:
nit: this could use a doccomment explaining the order in which `v`, `p_at_r` and `f_at_r` are expected to appear in the `Vec<F>`

Reply:
Done.